docker/src/main/java/brooklyn/entity/container/docker/DockerContainerImpl.java

```diff
@@ -606,7 +607,12 @@ public void deleteLocation() {
     }
 
     @Override
-    public void start(Collection<? extends Location> locations) {
+    public void start(Collection<? extends Location> locs) {
+        if (locs != null && !locs.isEmpty()) {
```

Comment:
this `if` check is no longer necessary - `addLocations` has been changed to accept null/empty and do no-op.